src/types.ts

```diff
@@ -13,38 +13,25 @@ export interface RouteConfigurationType {
   component: React.ReactNode;
 }
 
-export interface RequestUser {
-  username: string;
-  password: string;
-}
-
-export interface RequestBody {
-  client_id: string;
-  client_secret: string;
-  audience: string;
-  scope: string;
-  grant_type?: GrantType;
-  [key: string]: any;
-}
-
 export type MethodType = 'POST' | 'GET' | 'DELETE';
 
 export type AuthenticationResponseType = {
-  access_token: string;
-  id_token: string;
-  refresh_token?: string;
-  scope: string;
-  expires_in: number;
-  token_type: string;
+  accessToken: string;
+  idToken: string;
 };
 
-export type AccessTokenDecodeType = {
-  aud: string[];
-  azp: string;
-  exp: number;
-  gty: string;
-  iat: number;
+export type tokenDecodeType = {
```

Comment:
Types' names must be PascalCase. This type actually represents token's claims so choose more meaningful name.